resources/lang/nl.json

```diff
@@ -243,7 +243,7 @@
     "Enter URL": "URL invoerern",
     "Entries": "Entries",
     "Entries successfully reordered": "Entries succesvol herschikt.",
-    "Entry": "Binnenkomst",
+    "Entry": "Entry",
```

Comment:
Maybe this is an "Artikel / Bericht / Post" ?

Reply:
I’ve thought about this but those options exclude other possibilities or might confuse. Say you have a collection with products. Or reviews. Artikel or Bericht would be weird en Post implies mail. It’s difficult, so that’s why I went the English term which kinda works in Dutch. But if there’s a better alternative.

Reply:
Maybe "inzending / inzendingen" works but I agree, there is no real term in Dutch for this and we tend to lend words from the English language more often.

Reply:
Cool, thanks guys. I'd opt for Entry right now then. There are some more kinda common words I've chosen not to translate.